src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -453,6 +453,17 @@ s! {
         pub aio_flags: ::__u32,
         pub aio_resfd: ::__u32,
     }
+
+    pub struct fanotify_event_info_pidfd {
+        pub hdr: ::fanotify_event_info_header,
+        pub pidfd: ::__s32,
+    }
+
+    pub struct fanotify_event_info_error {
+        pub hdr: ::fanotify_event_info_header,
+        pub error: ::__s32,
+        pub error_count: ::__u32,
+    }
```

Comment:
Why are these GNU-only? I don't see them reexported in glibc, only uapi

Reply:
it s a bit of a mess on musl. with glibc it s enough to include `sys/fanotify.h` which in turn include `linux/fanotify.h` so you have all you need for this. in musl `sys/fanotify.h` which does not include `linux/fanotify.h` and if you do include both there are couple of type redefinition conflicts.